benches/src/lib.rs

```diff
@@ -0,0 +1,47 @@
+#![cfg_attr(windows, feature(abi_vectorcall))]
+#![allow(
+    clippy::must_use_candidate,
+    clippy::missing_panics_doc,
+    clippy::needless_pass_by_value,
+    clippy::implicit_hasher
+)]
+
+use ext_php_rs::{prelude::*, types::Zval};
+
+#[php_function]
+pub fn bench_function(n: u64) -> u64 {
+    // A simple function that does not do much work
+    n
+}
+
+#[php_function]
+pub fn bench_callback_function(callback: ZendCallable, n: usize) {
+    // Call the provided PHP callable with a fixed argument
+    start_instrumentation();
+    for i in 0..n {
+        callback
+            .try_call(vec![&i])
+            .expect("Failed to call function");
```

Comment:
Is there a more graceful way to report an error here instead of panicking for the benchmark?

Reply:
Well, if it fails to perform the operation it should fail. A panic is therefore valid imo